recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ outputs:
     script: install_nvcc.sh
     build:
       script_env:
-        - CUDA_HOME
+        - PATH
```

Comment:
Is there a reason not to still use `CUDA_HOME` here?

Reply:
@jakirkham All `CUDA_HOME` logic is pushed into the activate script at this point which I believe automatically inherits the environment variables, yes?

Reply:
Ah right. Wrong question. Why is `PATH` inherited?

Reply:
Because CUDA installation instructions say to add `$CUDA_INSTALL_DIR/bin` to `$PATH` to allow for calling things like `nvcc` or `cuda-memcheck`. Otherwise, we'd have to go on a hunt to find the `nvcc` or `cuda-memcheck` executables.

Reply:
See here: https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#environment-setup